internal/v4/search.go

```diff
@@ -28,7 +25,14 @@ func (h *Handler) serveSearch(_ http.Header, req *http.Request) (interface{}, er
 	if err != nil {
 		return "", err
 	}
-	h.groupsFromRequest(&sp, req)
+	admin, attrs, err := h.checkRequest(req)
```

Comment:
I don't actually want to error here, just carry on as if there is no logged in user, which there probably isn't, I have added a comment explaining this.